raiden/app.py

```diff
@@ -50,22 +50,11 @@ class App(object):  # pylint: disable=too-few-public-methods
         'shutdown_timeout': DEFAULT_SHUTDOWN_TIMEOUT,
     }
 
-    def __init__(self, config, chain, default_registry, discovery, transport_class=UDPTransport):
+    def __init__(self, config, chain, default_registry, discovery, transport=UDPTransport):
+        transport = self.transport_instantiation(self, config, transport)
```

Comment:
You will probably need to move this out of the constructor, as in `pass it as a constructor argument` or `inject the transport through the constructor`, e.g.:

```python
def __init__(self, config, chain, default_registry, discovery, transport):
   self.transport = transport
   ...
```

Reply:
Ah I see, thanks I will give that a try. I really appreciate the help! Sorry if I'm clogging up the commit history I will squash it all when I'm done.